src/components/BarChart.vue

```diff
@@ -54,22 +72,24 @@ a {
   justify-content: space-around;
   align-self: flex-end;
   flex: 1;
+  
 
-
-  background-color: grey;
+  /* background-color: rgb(206, 241, 222); */
 }
 
-.bar-chart__x__element {
-  padding: 0 12px;
-}
 
 .bar-chart__y {
   display: flex;
   flex-direction: column-reverse;
-  align-self: flex-end;
+  justify-content: space-between;
+  padding-block-end: 50px;
+  /* background-color: rgb(206, 241, 222); */
+}
 
+/* .bar-chart__y__element ['0'] {
+  background-color: green;
+
+} */

```

Comment:
Avoid committing commented out code. If it's commented out it usually means it will never be used again, so it just ends up cluttering the code and confusing devs.

Reply:
OK, so can you choose what code you keep from my pull request? In case you don't like my suggestion.

Reply:
I could checkout the branch you are basing your PR on and make the changes, but  since it's a fork from your project I'd have to do it in your fork. But in my experience the PR owner generally pushes up the changes requested after a review in order to get their "request" accepted.

I'll add you as a collaborator here though so you don't have to maintain your own fork. And then you can just clone and push to this repo.